### PR TITLE
Wounds UI Injury class type

### PR DESCRIPTION
**Pull request by Lemon-King**
Added injury class in core

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/CUModSquad/cu-core/pull/2" target="_blank">ReviewNinja

File: src/tsd/tsd.d.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/// <reference path="../../node_modules/definition-header/dist/index.d.ts" />
+/// <reference path="../../node_modules/ntypescript/bin/ntypescript.d.ts" />
```
Comment (codecorsair) [inline src/tsd/tsd.d.ts]: delete the ntypescript.d.ts line

Comment (Lemon-King) [inline src/tsd/tsd.d.ts]: !fixed

**Comment by codecorsair** [comment]
Add this to the new camelot-unchained library.  https://github.com/CUModSquad/Camelot-Unchained-Client-Library

**Pull request closed without merging**